scripts/darwin.sh

```diff
@@ -65,23 +71,28 @@ for disk in $DISKS; do
     done
   fi
 
-  echo "raw: $raw_device"
-
   if [[ "$protected" == "Yes" ]]; then
-    echo "protected: True"
+    echo "isReadOnly: True"
   else
-    echo "protected: False"
+    echo "isReadOnly: False"
   fi
 
   if [[ "$device" == "/dev/disk0" ]] || \
      [[ ( "$removable" == "No" ) || ("$removable" == "Fixed") ]] || \
      [[ ( "$location" =~ "Internal" ) && ( "$removable" != "Yes" ) && ( "$removable" != "Removable" ) ]] || \
      echo "$mountpoints" | grep "^/$"
   then
-    echo "system: True"
+    echo "isSystem: True"
   else
-    echo "system: False"
+    echo "isSystem: False"
   fi
 
+  echo "isVirtual: null"
+  echo "isRemovable: null"
```

Comment:
Maybe we should default this to `!isSystem`?

Reply:
Yes, good point.